app/controllers/advertisements_controller.rb

```diff
@@ -0,0 +1,28 @@
+class AdvertisementsController < ApplicationController
+  def index
+    @advertisement = Advertisement.all
```

Comment:
I'm assuming your are saying that all the instances of  of the `@advertisement` should be `@advertisements`